src/Sarif.Viewer.VisualStudio/Fixes/DisposableDifferenceViewerControl.xaml

```diff
@@ -0,0 +1,10 @@
+﻿<UserControl x:Uid="DifferenceViewerControl" 
+             x:Name="DifferenceViewerControl"
+             x:Class="Microsoft.Sarif.Viewer.Fixes.DisposableDifferenceViewerControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             mc:Ignorable="d">
+    <StackPanel x:Name="StackPanelContent"/>
+</UserControl>
```

Comment:
>UserControl [](start = 2, length = 11)

Man, it sure would be nice to have a link to where we got the code. Or at least some hints.

Reply:
Discussed offline.

---
In reply to: [516033858](https://github.com/microsoft/sarif-visualstudio-extension/pull/239#discussion_r516033858) [](ancestors = 516033858)